results/17-LV-GenomicVariantTest.Rmd

```diff
@@ -0,0 +1,146 @@
+---
+title: "17 - LV Genomic Variant Test"
+author: "Sara Gosline"
+date: "`r Sys.Date()`"
+output:
+  html_document:
+    code_folding: hide
+    toc: true
+    toc_float: true
+---
+
+
+```{r setup, include=FALSE}
+knitr::opts_chunk$set(echo = TRUE)
+library(synapser)
+synLogin()
+library(tidyverse)
+library(feather)
+```
+
+# Get Data
+
+This is nearly entirely based on the code in notebook 09 and that in 11. 
+
+We have latent variable expression analysis data
+- [Latent Variable Feather File](https://www.synapse.org/#!Synapse:syn21046991/tables/)
+
+For this data we are also using any data for which there are gene variants (cNFs, pNFs, MPNSTs):
+- [Exome-Seq variants](https://www.synapse.org/#!Synapse:syn20554939/tables/)
+- [WGS Variants](https://www.synapse.org/#!Synapse:syn20551862/tables/) 
+
+Let's see if there are any LVs that split based on gene variant. Because we're having trouble scaling with the number of latent variables, I only look at variants that occur in less than 5% of the population. notice this is a difference from notebook #11.
+
+```{r,warning=FALSE}
+wgs.vars=synTableQuery("SELECT Hugo_Symbol,Protein_position,specimenID,IMPACT,FILTER,ExAC_AF FROM syn20551862")$asDataFrame()
+exome.vars=synTableQuery("SELECT Hugo_Symbol,Protein_position,specimenID,IMPACT,FILTER,ExAC_AF FROM syn20554939")$asDataFrame()
+
+all.vars<-rbind(select(wgs.vars,'Hugo_Symbol','Protein_position','specimenID','IMPACT','ExAC_AF'),
+    select(exome.vars,'Hugo_Symbol','Protein_position','specimenID','IMPACT','ExAC_AF'))%>%
+  subset(ExAC_AF<0.05)
+
+
+fn <- tempfile(pattern = "", fileext = ".feather")
```

Comment:
this is now available as a table here: https://www.synapse.org/#!Synapse:syn21046991/tables/

If you update the code to include this, then I can add CHOP pNFs and it should be easy to rerun this Rmd and get the new data

Reply:
fixed.